providers/celery/README.rst

```diff
@@ -75,7 +75,8 @@ You can install such cross-provider dependencies when installing from PyPI. For
 Dependent package                                                                                                       Extra
 ======================================================================================================================  ===================
 `apache-airflow-providers-cncf-kubernetes <https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes>`_  ``cncf.kubernetes``
+`apache-airflow-providers-fab <https://airflow.apache.org/docs/apache-airflow-providers-fab>`_                          ``fab``
```

Comment:
This isn't right -- Celery shouldn't depend on FAB after #49031 was merged

Reply:
Let me rerun precommit on all files to see if it's resolved

Reply:
fixed. it has fab as optional dependency

Reply:
fixed